docs/source/quickstart.rst

```diff
@@ -59,6 +59,9 @@ Starting your project
 	# export the PDK variant depending on your shuttle, if you don't know leave it to the default
 	export PDK=sky130B
 
+    # for the GFMPW shuttles...
```

Comment:
can we make sure this is referenced from the main page as well: https://caravel-user-project.readthedocs.io/en/latest/ ?